bootstrap_views_generator.gemspec

```diff
@@ -36,7 +36,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'railties', '>= 4.0', '<= 7'
+  spec.add_runtime_dependency 'railties', '>= 7'
```

Comment:
Would you mind making this just `>= 6` in order to still support people on rails6. That should allow for supporting v6 and v7.